AutonomousDemo/Loop.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AutonomousDemo
+{
+    public class Loop
+    {
+
+        public void GameLoop(bool answer)
+        {
+            int lastTime = DateTime.Now.Millisecond;
```

Comment:
Use of real time is something that would happen in a video game, where you want the simulation to track with a person's perception of the passage of time. It has no purpose for you.